pyopenjtalk/__init__.py

```diff
@@ -41,41 +29,6 @@
 _global_htsengine = None
 
 
-# https://github.com/tqdm/tqdm#hooks-and-callbacks
-class _TqdmUpTo(tqdm):  # type: ignore
-    def update_to(self, b=1, bsize=1, tsize=None):
-        if tsize is not None:
-            self.total = tsize
-        return self.update(b * bsize - self.n)
```

Comment:
Why is the tqdm-related code removed? If it doesn't cause any problem, could you revert it back?

Reply:
It is still in steup.py and this removing is part of the

> Consider removing the lazy downloader in `pyopenjtalk/__init__.py`

I said yesterday.

Reply:
I meant why the following code:

```py
with TqdmUpTo(...):
  urlretrieve(...)
```

is reduced to

```py
urlretrieve(...)
```

I think the former was okay as is.

Reply:
Well, I just noticed that this code was removed before my changes by @synodriver . In my opinion, since this tqdm is only appears in setup time, whether removing or remaining it is okey. So what is your purpose of removing this?😂

Reply:
It's just a progress bar and not necessary. Besides, I noticed some wired issue about downloading when runing ci with tqdm. I have no idea what's happening but to remove it. Then the ci works fine.

Reply:
That may be an acceptable reason of removing this. Or we can pass a env variable to disable it in CI and enable it by default when user want to compile it natively?

Reply:
I have never had issues with tqdm on CI so far. Is the problem reproducible? If so, it's okay to remove it.

Reply:
I have run it several times on my fork but it always fail to build with tqdm installed. It's just like... the stdout is hiden, making it difficult to debug. And without tqdm it works fine. BTW, why does the progress bar so important for this project?

Reply:
Umm, OK. The progress bar was just useful for knowing the estimated time to finish the downloading process.

Reply:
But with prebuild-wheels, there is no need for users to download. You can check the [files](https://github.com/synodriver/pyopenjtalk/actions/runs/2847869617) I mentioned here, the size of them shows that they already contains data files in the wheel.